doc/articles/features/windows-system-profile.md

```diff
@@ -0,0 +1,38 @@
+﻿# Uno Support for Windows.System.Profile APIs
+
+## `AnalyticsInfo`
+
+### `DeviceForm` property
+
+The `DeviceForm` property now returns string from the following set:
+
+- Mobile
+- Tablet
+- Television
+- Car
+- Watch
+- VirtualReality
+- Desktop
+- Unknown
+
+In case of `WebAssembly` this property currently always returns `Unknown` as there is no reliable way to check for device form factor via JavaScript.
+
+## `VersionInfo.DeviceFamily`
+
+This string is in the form of `OSType.DeviceForm` where `OSType` is:
+
+- **Android** - `Android`
+- **iOS/iPadOS** - `Apple` 
```

Comment:
Why `Apple` and not `iOS`?

Reply:
Reason was that because of the new `iPadOS`, but probably could be easier to keep it just as `iOS` as iPadOS is still just iOS anyway 😀

Reply:
Right, vs say `MacOS` which *is* a completely different OS